executor/src/constant_evaluator/mod.rs

```diff
@@ -21,7 +19,7 @@ use rayon::prelude::{IntoParallelIterator, ParallelIterator};
 mod data_structures;
 
 pub const MIN_DEGREE_LOG: usize = 5;
-pub const MAX_DEGREE_LOG: usize = 10;
+pub const MAX_DEGREE_LOG: usize = 22;
```

Comment:
Now that witness generation doesn't pick the largest available size anymore, we can introduce this to a number we'll likely need in realistic (e.g. RISC-V) proofs.

There is still a cost to increasing that number: The fixed columns have to be computed and committed to in all sizes. That's more one-time cost for the prover. Also, the verification key becomes larger (but only parts of it are read by the verifier, so verification time remains unchanged).